api/app/elastic.js

```diff
@@ -1,17 +1,14 @@
 const items = require('../db/items.json')
 const elasticsearch = require('elasticsearch')
 
-const createClient = async () => {
-    return elasticsearch.Client({
-        hosts: [(process.env.ES_HOST || `http://localhost:9200`)],
-        maxRetries: 30,
-        requestTimeout: 30000
-    })
-}
+const client = elasticsearch.Client({
```

Comment:
What is the default connections limit of the connection pool?

Reply:
@jakubdyszkiewicz dont see any reference of the limit on the docs